### DISCUSSION TITLE
Confused what this library is for

### DISCUSSION THREAD

**Discussion by donnyv** [paviad/GoSharp#35] [General]
I'm new to the world of GO game programming. I was looking for a GO engine written in C# to use in a Unity Android app and found this project. Is this project just for creating GO protocol strings? or is it an engine? or just to see if a move is valid?

**Comment by paviad** [comment]
This library is used to manage a game tree with variations, it know about legal moves and the ko rule, and it knows about SGF file format.

**Comment by donnyv** [comment]
Do you know of any GO bots written in C#?

**Comment by paviad** [comment]
No, sorry.

**Comment by jiwoongim** [comment]
@donnyv  did you end up finding a good Go Engine for Unity?

**Comment by donnyv** [comment]
> @donnyv did you end up finding a good Go Engine for Unity?

Nope

**Comment by jiwoongim** [comment]
I see. Did you end up implementing one for yourself? What do you think about this one?

**Comment by donnyv** [comment]
I abandoned the project.
